public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -381,38 +381,23 @@ export class DashboardModel implements TimeModel {
   }
 
   variablesUpdated(newVariables: any) {
+    const rootStateKey = newVariables.get('dashboardUid').values;
```

Comment:
Actually I am now seeing a bug on this line... just investigating

Reply:
So actually I think the bug here was more on the audacity side... I fixed it by setting `dashboardUid` in the variables before we call the grafana POST function. So I think this PR can in fact be merged 🙂 

https://github.com/vivacitylabs/audacity/commit/88026aa78de283ca7802ac80dc4193d9fe1f9bbc